### PR TITLE
Edamam pipeline

### PR DESCRIPTION
**Pull request by Anggi-Permana-Harianja**
# Tell us what you do here

- [x] contributing new pipeline (please create an issue [here]() and link it below)
- [ ] implementing verified pipeline (please link a relevant issue labelled as `verified pipeline`)
- [ ] fixing a bug (please link a relevant bug report)
- [ ] improving, documenting, customizing existing pipeline (please link relevant issue)
- [ ] anything else (please link an issue or describe below)

# Relevant issue

issue #
[link](https://github.com/dlt-hub/pipelines/issues/95)

# More PR info

### CODE REVIEW

**Comment by Anggi-Permana-Harianja** [comment]
currently, implemented for duckdb

File: pipelines/edamam/__init__.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+import dlt
+from dlt.common.typing import TDataItem
+from dlt.extract.source import DltResource
+from dlt.sources.helpers import requests
+
+EDAMAM_NUTRITION_DATA_API_URL = 'https://api.edamam.com/api/nutrition-data'
+current_time = datetime.datetime.now()
+
+@dlt.source(name='edamam')
+def edamam_source(api_secrets_key: str = dlt.secrets.value) -> Sequence[DltResource]:
```
Comment (rudolfix) [inline pipelines/edamam/__init__.py:17]: I think for such a simple pipeline you do not even need to declare the source. it is perfectly fine to just use standalone resource as below, but if you declare it, allow to pass the full credentials and also the ingredients. but as said: if you expose the resource as below you just need one.

File: pipelines/edamam/__init__.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    """edamam dlt source"""
+    return edamam_source(api_secrets_key)
+
+@dlt.resource(write_disposition='append')
```
Comment (rudolfix) [inline pipelines/edamam/__init__.py:21]: append probably is OK as default

File: tests/edamam/test_edamam_pipeline.py [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    )
+    ingredients = ['100 gr chicken']
+    info = pipeline.run(nutrition_data(ingredients))
+    # test if jobs is loaded
+    assert_load_info(info)
+    # assert the schema
+    table = pipeline.default_schema.data_tables()
+    assert len(table) == 1
+    # assert that table is a dict
+    assert isinstance(table[0], dict)
```
Comment (rudolfix) [inline tests/edamam/test_edamam_pipeline.py:27]: the test is good. but you can reuse some building blocks. see tests/utils.py. there's a method that will return you row counts for tables. you also do not need to check the datatypes in the destination. do it by inspecting dlt schema, you already get table here. just check the columns.

**Review by rudolfix — CHANGES_REQUESTED** [review]
thx for contribution! see my review

**Comment by rudolfix** [comment]
@Anggi-Permana-Harianja as mentioned we are closing some of the community pipelines PRs. they will be moved to a separate repository or a sharing hub. when this is ready we'll resubmit them

**Pull request closed without merging**